index.html

```diff
@@ -133,5 +137,5 @@ <h4>Get In Touch</h4>
         </div>
     </footer>
 </body>
-
+    
```

Comment:
Why are these spaces here? A blank line is ok, but spaces?

Reply:
I have amended the above